subsys/net/l2/wifi/wifi_mgmt.c

```diff
@@ -335,3 +335,11 @@ static int wifi_set_power_save_timeout(uint32_t mgmt_request, struct net_if *ifa
 }
 
 NET_MGMT_REGISTER_REQUEST_HANDLER(NET_REQUEST_WIFI_PS_TIMEOUT, wifi_set_power_save_timeout);
+
+void wifi_mgmt_raise_twt_sleep_state(struct net_if *iface,
+				     int twt_sleep_state)
+{
+	net_mgmt_event_notify_with_info(NET_EVENT_WIFI_TWT_SLEEP_STATE,
+					iface, &twt_sleep_state,
```

Comment:
This certainly works but I just wonder if it would make sense here to send the value in the pointer itself using UINT_TO_POINTER() here and POINTER_TO_UINT() in the reading side. This way there would be no way to have a pointer that points to invalid memory when reading the value. WDYT?

Reply:
Yeah, it's better to use for single data type when utilites are available. I will update accordingly.

Reply:
Made the changes